.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+dist: trusty
+language: node_js
+node_js:
+  - "stable"
+notifications:
+  hipchat:
+    rooms:
+      secure: l5TTd5JuPAW883PtcyaIBcJI9Chr9JpsZPQAEUBKAgIEwzuS6y7t5arlkS1PwH6gi1FADzYDf+OXSIou4GkTSrIetnBcT/SAgF0gBKgIhj+eRkuCfZ4VaC7BPhfZ0hgYRE+5Ejf5BM2MJafRm0pj7OlqG4xKrQZwtuV1te5r3JY=
```

Comment:
all copy pasted from hls, there are not actually any unit tests to run, but it would be nice to have it setup when we do.